cli-tests/package.json

```diff
@@ -13,6 +13,7 @@
   },
   "dependencies": {
     "@playwright/test": "^1.32.3",
+    "@types/shelljs": "^0.8.12",
```

Comment:
This can go to `devDependencies`

Reply:
I'm not sure really sure. Since this is a subproject implementing "tests" only. So it's original flow is "dev" only. And we would need to add and use "dev build" for tests. And we don't have `prod` and `dev` state for tests subprojects in our build process. Or I'm missing this?

Reply:
All dependencies which are not required to run the app are considered `devDependencies`.
Since `@types/*` is not required to run the app, it shall be moved to `devDependencies`

Reply:
Got it. I've missed to think from this point. Moved